Source/SelfService/Web/microservice/microserviceView/configuration/configuration.tsx

```diff
@@ -27,57 +19,28 @@ export type ConfigurationProps = {
     currentMicroservice: MicroserviceStore;
 };
 
-const styles = {
-    divider: {
-        backgroundColor: '#3B3D48'
-    }
-};
-
-export const Configuration = ({
-    application, applicationId, environment, microserviceId, currentMicroservice }: ConfigurationProps) => {
-
-    const history = useHistory();
-
-    const handleEnvironmentClick = async () => {
-        const href = `/microservices/application/${applicationId}/${environment}/view/${microserviceId}/environment-variables`;
-        history.push(href);
-    };
-
-    return (
-        <>
-            <SetupSection
-                application={application}
-                applicationId={applicationId}
-                environment={environment}
-                currentMicroservice={currentMicroservice}
-                microserviceId={microserviceId}
-            />
-
-            <FilesSection
-                applicationId={applicationId}
-                environment={environment}
-                microserviceName={currentMicroservice.name}
-                microserviceId={microserviceId}
-            />
-
-            <Divider sx={styles.divider} />
-
-            <Box ml={2}>
-                <LiveIngressView urls={currentMicroservice.live.ingressUrls} paths={currentMicroservice.live.ingressPaths} />
-            </Box>
-
-            <Box ml={2}>
-                <ButtonText
-                    onClick={handleEnvironmentClick}>
-                    Manage environment variables
-                </ButtonText>
-
-                <DownloadButtons
-                    environment={environment}
-                    microserviceName={currentMicroservice.name}
-                    applicationId={applicationId}
-                />
-            </Box>
-        </>
-    );
-};
+export const Configuration = ({ application, applicationId, environment, microserviceId, currentMicroservice }: ConfigurationProps) => (
```

Comment:
The `Configuration`-component now looks like a good container component that deals with layout and structure of the page. On that note I suggest moving the `Accordion` components outside the different `*Section`-components. 

Doing so will also give the container page the ability to make decisions to example, only expand a single accordion at a time (if that is ever a need).

ex: 
```
<Accordion {...props }>
    <SetupSection  {...props } /> 
</Accordion>

<Accordion {...props }>
    <FilesSection  {...props } /> 
</Accordion>

//etc...
```